### ISSUE TITLE
Print or Export to PDF

### ISSUE THREAD

**Issue by jbnv** [Roam-Research/issues#456]
I currently don't see any way to print a Roam page or export a page in a manner so that it can be printed easily. 

* I tried exporting a document to Markdown. This required converting the document to HTML, an unnecessarily extra step to begin with. I found that all of the non-Markdown markup in the document was not rendered correctly, defeating the purpose of going through this step.

* I tried printing the page from the browser itself. The browser rendered only the first page and included the sidebar. 

I'm using Roam to work on a project where I will frequently work off computer. While Roam is solving several problems for me, this inability to print is a major drawback.

**How would the feature look like to you, and how would it fit in with your interaction patterns?** It doesn't have to be anything fancy, just another option in the top-right menu or a PDF option in the Export command. Alternatively, making use of the browser's Print command or an export to HTML would work.

**Comment by gabrielmontagne** [comment]
I agree.  I think a print only CSS would be enough.  Probably a 3 line CSS  tweak of overflow, just for print, might make this suddenly work?
